src/NServiceBus.Core/Outbox/OutboxDeduplicationBehavior.cs

```diff
@@ -63,23 +58,35 @@ void DispatchOperationToTransport(IEnumerable<TransportOperation> operations)
 
                 deliveryOptions.EnlistInReceiveTransaction = false;
 
-                var message = new OutgoingMessage(transportOperation.MessageId,transportOperation.Headers, transportOperation.Body);
+                var message = new OutgoingMessage(transportOperation.MessageId, transportOperation.Headers, transportOperation.Body);
 
-                //dispatch to transport
+                var operationType = transportOperation.Options["Operation"];
```

Comment:
I also have a hunch that we might backtrack, but the inheritance was getting in the way. 

A plan to get rid of the "DeliveryOptions" and friends and then we can reevalute and perhaps put in a better abstraction?